h2o-genmodel/src/main/java/hex/genmodel/algos/gbm/GbmMojoModel.java

```diff
@@ -51,4 +52,33 @@ public GbmMojoModel(String[] columns, String[][] domains) {
     public double[] score0(double[] row, double[] preds) {
         return score0(row, 0.0, preds);
     }
+
+    /**
+     * Calculate number of compressed trees per class
+     * @return number of compressed trees per class
+     */
+    public int calcNClassesToScore() {
+        int nClassesToScore;
+
+        if (_family == bernoulli || _family == modified_huber) {
+            nClassesToScore = 1;
+        }
+        else if (_family == multinomial) {
+            nClassesToScore = _nclasses == 2 ? 1 : _nclasses;
+        }
+        else {
+            nClassesToScore = 1;
+        }
+
+        return nClassesToScore;
+    }
```

Comment:
Looking at GBM.java#L104, I see that if `_family` is `bernoulli | modified_huber` then number of classes in the model is always 2. And if `_family` is `multinomial` then number of classes can be anything, and for all other families number of classes is 1.

Thus this function can be simplified to `return _nclasses == 2 ? 1 : _nclasses;`, which is exactly what the original expression was...